### ISSUE TITLE
Make # of Poles & # of Throws editable on Relay(s) -and- add "contactor" as lookup term

### ISSUE THREAD

**Issue by JP-Higgs** [circuitdiagram/components#214]
I commonly use multi-pole and multiple throws on my relays and see no built-in straight-forward way to do this as is. If the option to edit the number of each, this would avoid having numerous items added yet cover a large amount of options.
Additionally, adding the term "contactor" to the searchable list for relays will make finding those components easier for our friends overseas (and a few in the states as well).

**Comment by whatsreal** [comment]
Agreed.  I used dual-pole dual-throw relays in my course labs.  It would be helpful to have.

**Comment by theimportedking** [comment]
Contactors are heavily used in the trades. +1 for editable relays. 3pst is pretty common.
